src/sdl2/rect.rs

```diff
@@ -67,6 +67,7 @@ fn clamped_mul(a: i32, b: i32) -> i32 {
 /// recommended to use `Option<Rect>`, with `None` representing an empty
 /// rectangle (see, for example, the output of the
 /// [`intersection`](#method.intersection) method).
+#[repr(transparent)]
```

Comment:
repr(transparent) should not be necessary, and I would even say it should be avoided in both Rect and Point.

Reply:
the functions `raw` and `raw_slice` cast between the types (on both Point and Rect), so they need to be layout compatible